Apps/W1/HybridGP/app/src/pages/IntelligentCloudExtension.PageExt.al

```diff
@@ -31,6 +31,11 @@ pageextension 4015 "Intelligent Cloud Extension" extends "Intelligent Cloud Mana
 
     actions
     {
+        modify(RunDataUpgrade)
+        {
+            Visible = UseTwoStepProcess;
```

Comment:
I would leave this action as visible, no need to hide it. We would need to run it manually if the task is not started

Reply:
it is not guaranteed that the task will start

Reply:
I'll remove this change.